verifier/src/main/java/com/exasol/classlistextractor/verifier/ClassListServer.java

```diff
@@ -161,17 +167,22 @@ public void run() {
         private List<String> readFromClientBlocking(final Socket client) {
             try (final InputStream inputStream = client.getInputStream()) {
                 final String classList = new String(inputStream.readAllBytes(), StandardCharsets.UTF_8);
-                return Arrays.asList(classList.split("\n"));
+                final List<String> classNames = Arrays.asList(classList.split("\n"));
+                LOGGER.fine(() -> "Read " + classNames.size() + " class name from client");
+                return classNames;
             } catch (final IOException exception) {
                 throw new UncheckedIOException(ExaError.messageBuilder("E-JCLE-VF-5")
                         .message("Failed to read from class list client.").toString(), exception);
             }
         }
 
         private static class ServerThreadMonitor {
-            @Getter
             private boolean noMoreClients = false;
 
+            public boolean isNoMoreClients() {
```

Comment:
Would it be possible to use a positive wording here? 
```suggestion
            private static class ServerThreadMonitor {
            private boolean acceptClients = true;

            // rename to isAcceptClients() if the wording sounds to similar to you
            public boolean acceptsClients () { 
                return acceptClients ;
            }

            public void acceptClients (boolean value) {
                this.acceptClients = value;
            }
        }
```